src/lucky/response.cr

```diff
@@ -1,9 +1,9 @@
 class Lucky::Response
   DEFAULT_STATUS = 200
 
-  getter context, content_type, body
+  getter context, content_type, body, debug_message
 
-  def initialize(@context : HTTP::Server::Context, @content_type : String, @body : String, @status : Int32? = nil)
+  def initialize(@context : HTTP::Server::Context, @content_type : String, @body : String, @status : Int32? = nil, @debug_message : String? = nil)
```

Comment:
These arguments are getting a little log, but I'm not sure how best to handle this in Crystal.  In Ruby, I'd probably move the optional args into an options Hash.  Is there something similar to be done here?

Reply:
I would put @context on a newline and then run `crystal tool format` and it should put each of them on a newline so it's a bit easier to read and isn't so wide. LMK if that does the trick :)

Reply:
The formatter only seems to enforce proper indentation in this case.
<img width="1218" alt="screen shot 2017-12-19 at 7 08 05 pm" src="https://user-images.githubusercontent.com/9160106/34186226-00ba433e-e4f0-11e7-8e09-26ad1990ed41.png">

The formatter also insists that the closing paren be on the same line as the last argument.  That would normally mean that the opening paren should be on the same line as the first argument, but the formatter doesn't enforce that.  

It does still let me manually make the change though:
<img width="460" alt="screen shot 2017-12-19 at 7 14 02 pm" src="https://user-images.githubusercontent.com/9160106/34186386-cd9c467c-e4f0-11e7-854d-cca7a9dc2e00.png">

Reply:
I just ran into this myself. Strange that it doesn't automatically put them on newlines. I like what you have in that second screenshot though 👍